themes/website/static/js/osdetect.js

```diff
@@ -84,9 +84,23 @@ if (window.navigator.userAgent.indexOf("Linux") != -1) {
 	OSName = "Linux";
 
 	clients = [{
-		title: "Tox repository",
+		title: "Debian",
 		name: "repo",
-		icon: "list-ul",
+		icon: "icon-debian",
+		desc: false,
+		dlLink: "#gnulinux",
+	},
```

Comment:
Another nitpick: in some places in this file, the braces between structs are done like so: `}, {` without the endline after the first brace. It looks like the `}, {` style was previously used, so you should probably stick to that.